frontend/public/components/image-stream.tsx

```diff
@@ -52,14 +52,31 @@ const ImageStreamTagsRow: React.SFC<ImageStreamTagsRowProps> = ({imageStream, sp
 
 export const ImageStreamsDetails: React.SFC<ImageStreamsDetailsProps> = ({obj: imageStream}) => {
   const imageRepository = _.get(imageStream, 'status.dockerImageRepository');
+  const publicImageRepository = _.get(imageStream, 'status.publicDockerImageRepository');
+  const imageCount = _.get(imageStream, 'status.tags.length');
   const specTagByName = _.keyBy(imageStream.spec.tags, 'name');
 
   return <div>
     <div className="co-m-pane__body">
-      <ResourceSummary resource={imageStream} showPodSelector={false} showNodeSelector={false}>
-        {imageRepository && <dt>Image Repository</dt>}
-        {imageRepository && <dd>{imageRepository}</dd>}
-      </ResourceSummary>
+      <div className="row">
+        <div className="col-sm-12 col-md-6">
```

Comment:
Consider only adding col-md-6 class when there is a `publicImageRepository`. This shouldn't be too difficult using `classNames`